src/test/java/AllTests.java

```diff
@@ -0,0 +1,12 @@
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import com.everis.boundary.CarResourcesTest;
+import com.everis.control.CarServiceTest;
+
+@RunWith(Suite.class)
+@SuiteClasses({CarServiceTest.class, CarResourcesTest.class })
```

Comment:
This class is not necessary, maven and the IDE automatically run all the tests for us 😄